dash/development/_all_keywords.py

```diff
@@ -0,0 +1,72 @@
+# This is a set of Python keywords that cannot be used as prop names.
+# Keywords for a particular version are obtained as follows:
+# >>> import keyword
+# >>> keyword.kwlist
+
+kwlist = set([
```

Comment:
We should change this to `python_keywords` then to be consistent. I think we should do it here, it's only 4 lines and I don't think it needs its own PR.